components/sso-saml/org.wso2.carbon.identity.sso.saml.ui/src/main/resources/org/wso2/carbon/identity/sso/saml/ui/i18n/Resources.properties

```diff
@@ -53,6 +54,7 @@ sp.enter.not.valid.endpoint.address=Please enter valid URL for assertion consume
 sp.enter.valid.endpoint.address=Please enter at least one valid URL for assertion consumer URLs
 sp.endpoint.address.already.exists=Assertion consumer URL entered is already exists
 sp.enter.default.acs=Please select default assertion consumer URL
+sp.enter.default.signingAlgorithm = Please select default response Signing algorithm
```

Comment:
Done. Deleted 'default' as it also confuses the user. We have the default set in the identity.xml.